pyface/ui/qt4/tasks/split_editor_area_pane.py

```diff
@@ -173,38 +173,50 @@ def set_layout(self, layout):
     ##########################################################################
 
     def get_context_menu(self, pos):
-        """ Returns a context menu containing split/collapse actions
+        """ Return a context menu containing split/collapse actions.
 
-        pos : position (in global coordinates) where the context menu was
-        requested
+        Parameters
+        ----------
+        pos : QtCore.QPoint
+            Mouse position in global coordinates for which the context menu was
+            requested.
+
+        Returns
+        -------
+        menu : pyface.action.menu_manager.MenuManager or None
+            Context menu, or None if the given position doesn't correspond
+            to any of the tab widgets.
         """
         menu = MenuManager()
-        splitter = None
 
-        splitter = None
         for tabwidget in self.tabwidgets():
-            # obtain tabwidget's bounding rectangle in global coordinates
-            global_rect = QtCore.QRect(tabwidget.mapToGlobal(QtCore.QPoint(0, 0)),
-                                        tabwidget.size())
-            if global_rect.contains(pos):
+            widget_pos = tabwidget.mapFromGlobal(pos)
+            if tabwidget.rect().contains(widget_pos):
                 splitter = tabwidget.parent()
-
-        # no split/collapse context menu for positions outside any tabwidget
-        # region
-        if not splitter:
-            return
+                break
+        else:
+            # no split/collapse context menu for positions outside any
+            # tabwidget region
+            return None
 
         # add split actions (only show for non-empty tabwidgets)
         if not splitter.is_empty():
-            actions = [Action(id='split_hor', name='Create new pane to the right',
-                       on_perform=lambda : splitter.split(orientation=
-                        QtCore.Qt.Horizontal)),
-                       Action(id='split_ver', name='Create new pane to the bottom',
-                       on_perform=lambda : splitter.split(orientation=
-                        QtCore.Qt.Vertical))]
-
-            splitgroup = Group(*actions, id='split')
-            menu.append(splitgroup)
+            split_group = Group(
```

Comment:
Changes in this block are purely stylistic, except that I've changed "Create new pane to the bottom" to the more natural sounding (at least to me) "Create new pane below".